components/module-components/infinitie-carousel/InfiniteCarousel.tsx

```diff
@@ -7,8 +7,9 @@ const InfiniteCarousel = () => {
   return (
     <div className={style.slider}>
       <div className={style.slideTrack}>
-        {multipliedArray.map((item) => (
-          <div className={style.slide} key={Math.floor(Math.random() * 1000)}>
+        {multipliedArray.map((item, index) => (
+          // eslint-disable-next-line react/no-array-index-key
+          <div className={style.slide} key={index}>
```

Comment:
Since the items are just strings, you can use the item itself as a key.
``` 
{multipliedArray.map((item) => (
          <div className={style.slide} key={item}>
              {item}
          </div>
        ))}
```